assets/js/script.js

```diff
@@ -4,126 +4,175 @@
  * TODO: Add support for subtraction
  */
 const OPERATIONS = {
-    "addition": {"symbol": "+", "createQuestion": createAdditionQuestion},
-    "multiplication": {"symbol": "x", "createQuestion": createMultiplicationQuestion},
-    "division": {"symbol": "÷", "createQuestion": createDivisionQuestion},
+  addition: { symbol: "+", createQuestion: createAdditionQuestion },
```

Comment:
Ah, cool. Yeah, I'm very much in json land and wasn't think of this as a pure js object 🙃